core/server/api/authentication.js

```diff
@@ -327,7 +327,7 @@ authentication = {
                     return updatedUser.save(options);
                 })
                 .catch(common.errors.ValidationError, (err) => {
-                    return Promise.reject(err); 
+                    return Promise.reject(err);
```

Comment:
Why is this happening? 🤔

Reply:
We had an extra space after semi-colon in this code, came with merge of `eslint-plugin-ghost` changes it seems, apparently lint didn't catch and I inadvertently fixed it while updating this file 🙂

Reply:
Ok nice.

Looks like https://eslint.org/docs/rules/no-trailing-spaces is not defined in our plugin 🤔 Worth raising an issue. You wanne do it?

Reply:
Yeah on it 🏃